src/lib/transaction.ts

```diff
@@ -120,11 +124,9 @@ export async function createTransaction<
           } as Types.CreateTransactionReturnProps<T>;
         }
       } else {
-        const Bundlr = await import(
-          '@bundlr-network/client/build/esm/node/bundlr'
-        );
-        const bundlr = new Bundlr.default(
-          'http://node2.bundlr.network',
+        const { NodeBundlr } = await import('@bundlr-network/client');
```

Comment:
`NodeBundlr` is once again throwing the following `SubtleCrypto` errors which is why the import was changed to `const Bundlr = await import( '@bundlr-network/client/build/esm/node/bundlr' );`.

The error received is as follows:

```bash
Error: SubtleCrypto not available!
    at new WebCryptoDriver (C:\Users\rohit\Desktop\Coding Temp\arweavekit\node_modules\@bundlr-network\client\node_modules\arweave\web\lib\crypto\webcrypto-driver.js:10:19)
    at Object.<anonymous> (C:\Users\rohit\Desktop\Coding Temp\arweavekit\node_modules\@bundlr-network\client\node_modules\arweave\web\common.js:120:18)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (C:\Users\rohit\Desktop\Coding Temp\arweavekit\node_modules\@bundlr-network\client\node_modules\arweave\web\index.js:17:18)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
```
This is a known incompatibility between `Bundlr` and the `arweave` version it uses.

Reply:
@ropats16 Please check it out here [https://github.com/pawanpaudel93/arweavekit-examples-test](https://github.com/pawanpaudel93/arweavekit-examples-test). Its working fine for me without subtlecrypto error.

Reply:
After updating some Vite configs in the test repo as suggested, this works locally as well 👍